.github/workflows/tests.yml

```
@@ -26,7 +26,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby-version: ['3.1.0', '3.0.1', '2.7.3', '2.6.7']
+        ruby-version: ['3.2.0', '3.1.0', '3.0.1', '2.7.3', '2.6.7']
```

Alternate if you want to go to MAJOR.MINOR only
```suggestion
        ruby-version: ['3.2', '3.1', '3.0', '2.7', '2.6']
```